.github/workflows/ci.yml

```diff
@@ -65,6 +65,44 @@ jobs:
                 build_py_project_in_conda_env
                 build_docs
 
-# vim: sw=4
+    downstream_tests:
+        strategy:
+            matrix:
+                # Reinstate when
+                # https://github.com/illinois-ceesd/mirgecom/issues/212
+                # is fixed:
+                # downstream_project: [meshmode, pytential, pytato, mirgecom]
 
+                downstream_project: [meshmode, pytential, pytato]
```

Comment:
meshmode and pytential aren't downstream of grudge?

Reply:
Ok, but which projects would you like to see tested here? I'm not sure that pytato is downstream of grudge either.

Reply:
Only mirgecom is downstream of grudge.

Reply:
Ok, so does it make sense to merge this PR, considering that illinois-ceesd/mirgecom#212 is still open?